src/Tizen.NUI/src/internal/Application.cs

```diff
@@ -1194,12 +1195,22 @@ public static Application New(int argc, string stylesheet, Application.WindowMod
 
         public static Application New(string[] args, string stylesheet, Application.WindowMode windowMode)
         {
-            int argc = args.Length;
-            string argvStr = string.Join(" ", args);
-
-            Application ret = new Application(NDalicPINVOKE.Application_New__MANUAL_4(argc, argvStr, stylesheet, (int)windowMode), true);
+            Application ret = null;
+            try
+            {
+                int argc = args.Length;
+                string argvStr = string.Join(" ", args);
 
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                ret = new Application(NDalicPINVOKE.Application_New__MANUAL_4(argc, argvStr, stylesheet, (int)windowMode), true);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+            catch (Exception exception)
+            {
+                Tizen.Log.Fatal("NUI", "[Error] got exception during Application New(), this should not occur, msg : " + exception.Message);
+                Tizen.Log.Fatal("NUI", "[Error] error line number : " + new StackTrace(exception, true).GetFrame(0).GetFileLineNumber());
```

Comment:
This won't work if there is no 'pdb' file. (I believe this is the case for `tizenfx`.)
Please break down the current try-catch block into pieces until you find the exact point of exception.

Reply:
Do you mean to separate the try-catch for each line?

Reply:
Fixed.